src/olympia/lib/settings_base.py

```diff
@@ -1382,7 +1397,7 @@ def get_db_config(environ_var, atomic_requests=True):
         },
         'z.task': {
             'handlers': ['mozlog'],
-            'level': logging.DEBUG,
+            'level': logging.INFO,
             'propagate': False
```

Comment:
I suppose this is the default config now (inherited from `root`) so we could probably remove this entry.